### PR TITLE
[plugin-tester] Custom init path

### PR DESCRIPTION
**Pull request by takecare**
Addresses #22 and checks if plugin is resolvable before starting server.

### CODE REVIEW

File: plugin-tester/src/index.js [hunk 1]
```diff
@@ -55,14 +55,26 @@ const runPlugin = (pluginPath, options) => {
     }
 }
 
+const pluginIsInPath = pluginPath => {
+    try {
+        require.resolve(pluginPath)
```
Comment (ouchadam) [inline plugin-tester/src/index.js:60]: oooh TIL

File: plugin-tester/src/index.js [hunk 2]
```diff
@@ -55,14 +55,26 @@ const runPlugin = (pluginPath, options) => {
...
     .description('runs the plugin in a local server')
     .option('-w, --watch', 'watch the source directory for changes')
     .option('-p --port', 'optional local server port. default 5000')
     .option('-c --config <path>', 'path to json representation of a configuration')
     .action((inputPath, options) => {
         const pluginPath = inputPath ? path.resolve(inputPath) : process.cwd()
-        runPlugin(pluginPath, options)
+        if (pluginIsInPath(pluginPath)) {
+            runPlugin(pluginPath, options)
+        }
```
Comment (ouchadam) [inline plugin-tester/src/index.js]: should we throw a nice message in the `false` case?

File: plugin-tester/src/template.js [hunk 3]
```diff
@@ -23,14 +23,15 @@ const template = (templatePath, viewState, file) => {
...
-    console.log("Creating plugin... ", name);
-
+const createTemplate = (name, customPluignPath) => {
     const viewState = {
         pluginName: name,
         pluginId: name.toLowerCase().replace(/\s+/g, '-')
     }
-    const pluginPath = path.join(process.cwd(), name)
+
+    const pluginPath = customPluignPath ? customPluignPath : path.join(process.cwd(), name)
```
Comment (ouchadam) [inline plugin-tester/src/template.js]: `customPluignPath` :ghost:

**Comment by ouchadam** [comment]
small `?` around whether we should throw after failing to resolve a plugin

File: plugin-tester/src/index.js (hunk 2 above)
Comment (takecare) [reply plugin-tester/src/index.js]: there's a message being printed in the method already. i'm moving it here instead since it makes more sense.

**Review by ouchadam — APPROVED** [review]

**Pull request merged**